### DISCUSSION TITLE
Trying to access Shadowsocks externally.

### DISCUSSION THREAD

**Discussion by NB-2022** [qdm12/gluetun#755] [Q&A]
Hello,

I am trying to access Shadowsocks externally for this type of set-up (User>Shadowsocks proxy on VPS>VPN). This is the config I have been unsuccessfully trying (no connection from the tablet to the Shadowsocks proxy), a direct Shadowsocks server not hosted within docker (directly connected without passing to VPN does work, on another port).

```
docker run -it --rm --cap-add=NET_ADMIN -e VPNSP=fastestvpn \
-e OPENVPN_USER=[username censored] -e OPENVPN_PASSWORD=[password censored] \
-e SERVER_HOSTNAME=[hostname censored] -e SHADOWSOCKS=on -e SHADOWSOCKS_ADDRESS=:5062 -e SHADOWSOCKS_CIPHER=chacha20-ietf-poly1305 -e SHADOWSOCKS_PASSWORD=[password censored] -p 5062:5062/tcp -p 5062:5062/udp --name [container name censored] qmcgaw/gluetun
```

Any idea how to access the Shadowsocks proxy remotely.

**Comment by qdm12** [comment]
> User>Shadowsocks proxy on VPS>VPN

That's what I use and it works even outside my home.
Maybe try surrounding the shadowsocks address environment variable value with double quotes? What are your logs regarding shadowsocks?

Also what's your shadowsocks client and what does it say?
